### DISCUSSION TITLE
libre 2 EU  change to  auto-readings

### DISCUSSION THREAD

**Discussion by Felguera242** [NightscoutFoundation/xDrip#3142] [Q&A]
Hi you all , i use libre 2 EU sensors  with xdrip and xwatch  with no problemas at all.
I have received a notification in my librelink app  ,announcing that when i change my next sensor , the app (librelink) will measure readings automatically every period of time (dont know how it works) ,  What i am afraid is the posibility that my setup with xdrip and xwatch (amazfit gtr2e)  will stop working with this change. Any news about it? 
Thanks uin advance and sorry if this issue is commented in the forum.

**Comment by gui-dos** [comment]
To decrypt the European Libre 2 data you just need the sensor UID (that you can scan via NFC), not the sensor `patchInfo` whose two latest bytes change at every new scan and that you have to encode and send to the sensor to unlock the BLE payload as they were at the time of the streaming activation.

So, yes, for the European Libre 2 it is possible to run another app in a “companion” mode with the official LibreLink app as for the Dexcom, like xDrip4iO5 is already allowing, albeit after activating a debugging option to skip to send the unlock payload.

**Reply by Felguera242** [reply]
Thanks for your awesome "technical answer" . So i suppose i wont have to change anything ,we´ll see soon , i will keep you all updated.
Thank you very much.

**Comment by AliQ101** [comment]
Compatibility with last libreLink #3069 ?
